docs/api/search.md

```diff
@@ -3,20 +3,28 @@ Search
 
 Our api supports searching artifact metadata which will return a link header for each search result.
 
-Note:
-* The only requirement is the search portion of the criteria.
 
-Search criteria:
-----------------
+Search criteria
+---------------
 
 * We currently support equality, wildcard, and version searches.
 * Equality search syntax: `"field=value"`
 * Wildcard search syntax: `"field=valu*"` where the `*` represents the 0 or more characters.
 * Version search syntax: `"field~=1.1"` where search results >= 1.1 but < 2.
-* We support escaping of \*, ~, and = for literal evaluation.
 
-Sort criteria:
---------------
+
+Escaping Special Characters
+---------------------------
+
+As you can see \*, ~, and = are special characters within search criteria. We support escaping of these characters for literal evaluation. These characters must be escaped with double backslashes to be valid JSON. Here is an example:
```

Comment:
Yes, so it renders right in markdown. Unless I am forgetting something, pretty sure I have to for that to render properly.